dpytools/http/dataset/base_dataset.py

```diff
@@ -0,0 +1,39 @@
+import json
+from typing import Dict
+
+from requests import RequestException, Response
+
+from dpytools.http.base import BaseHttpClient
+from dpytools.http.upload.token_auth import TokenAuth
+from dpytools.logging.logger import DpLogger
+
+logger = DpLogger("dpytools")
+
+
+class BaseDatasetClient(BaseHttpClient):
+    def __init__(self, url: str, backoff_max: int = 30):
+        super().__init__(backoff_max=backoff_max)
+        self.token_auth = TokenAuth(backoff_max=backoff_max)
+        self.url = url
+        self.dataset_url = f"{self.url}/dataset"
+
+    def send_json(self, url: str, json_data: Dict) -> None:
```

Comment:
Rename this to `def post_json` (in case we need to add other HTTP methods later)